widgetssdk/src/main/java/com/glia/widgets/engagement/EngagementRepositoryImpl.kt

```diff
@@ -148,9 +207,9 @@ internal class EngagementRepositoryImpl(
     @SuppressLint("CheckResult")
     override fun declineCurrentEngagementRequest() {
         _engagementRequest.firstOrError().flatMapCompletable(::decline).subscribe({
-            Logger.i(TAG, "Incoming Call Visualizer engagement was declined")
+            i(TAG, "Incoming Call Visualizer engagement was declined")
```

Comment:
I would suggest to keep the full reference: `Logger.d()`, `Logger.i()`, `Logger.e()`, like we have in the majority of places. Keeping single style makes search easier.